Transformer/main.swift

```diff
@@ -14,13 +14,19 @@
 
 import Python
 import TensorFlow
+import Foundation
 
+let modelName = "117M"
 let sys = Python.import("sys")
 sys.path = sys.path + ["."]
-let encoder = Python.import("encoder").get_encoder("117M")
-
-let checkpoint = "models/117M/model.ckpt"
-let model = TransformerLM(contentsOfPythonCheckpointFile: checkpoint, scope: "model")
+let encoder = Python.import("encoder").get_encoder(modelName)
+
+let checkpoint = "models/\(modelName)/model.ckpt"
+let configFile = "models/\(modelName)/hparams.json"
+let configData = try Data.init(contentsOf: URL(fileURLWithPath: configFile))
```

Comment:
```suggestion
let configData = try Data(contentsOf: URL(fileURLWithPath: configFile))
```

Use the initializer syntax instead of explicit `.init`.